### PR TITLE
Updated for Minecraft 1.12 / Forge Beta #2322

### PR DESCRIPTION
**Pull request by Rires-Magica**
GuiAccountSelector.java was rewrited by IDEA ><
Supported MC1.12 and added Jp langage file

### CODE REVIEW

File: src/main/java/the_fireplace/ias/config/IASGuiFactory.java [hunk 1]
```diff
@@ -15,7 +15,17 @@
 	@Override
 	public void initialize(Minecraft minecraftInstance) {}
 
-	@Override
+  @Override
```
Comment (BrainStone) [inline src/main/java/the_fireplace/ias/config/IASGuiFactory.java]: Fix intendation

File: src/main/java/the_fireplace/ias/gui/GuiAccountSelector.java [hunk 2]
```diff
@@ -29,374 +29,373 @@
...
-			}
-			int color = 16777215;
-			if (Minecraft.getMinecraft().getSession().getUsername().equals(data.alias))
-			{
-				color = 0x00FF00;
-			}
-			GuiAccountSelector.this.drawString(GuiAccountSelector.this.fontRendererObj, s, p_180791_2_ + 2, p_180791_3_ + 1, color);
-		}
-	}
+  private int selectedAccountIndex = 0;
```
Comment (BrainStone) [inline src/main/java/the_fireplace/ias/gui/GuiAccountSelector.java]: Fix intendation

File: src/main/java/the_fireplace/ias/gui/GuiPasswordField.java [hunk 3]
```diff
@@ -30,14 +30,14 @@ public boolean textboxKeyTyped(char typedChar, int keyCode)
...
-	public void mouseClicked(int mouseX, int mouseY, int mouseButton)
+	public boolean mouseClicked(int mouseX, int mouseY, int mouseButton)
 	{
 		// Minecraft has variable-width fonts, so replace the text with asterisks so that the correct cursor position is calculated
 		String password = getText();
 		replaceText(Strings.repeat('*', getText().length()));
 		super.mouseClicked(mouseX, mouseY, mouseButton);
 		replaceText(password);
-
+    return true;
```
Comment (BrainStone) [inline src/main/java/the_fireplace/ias/gui/GuiPasswordField.java]: Fix intendation

File: src/main/resources/assets/accswitcher/lang/ja_JP.lang [hunk 4]
```diff
@@ -0,0 +1,24 @@
+ias.loggedinas=あなたは次のアカウントでログインしています:
```
Comment (BrainStone) [inline src/main/resources/assets/accswitcher/lang/ja_JP.lang]: Make a separate PR for a new language

**Comment by The-Fireplace** [comment]
@BrainStone The indentation is fine, this time. I planned to have Idea fix all the indentation soon, anyways.
@Rires-Magica In the future, PRs with new language localizations should be kept seperate from other big changes. And indentation should be kept consistent.
